scripts/monitor_lambdas.py

```diff
@@ -75,28 +105,57 @@ def send_slack_post(webhook:str, stages:dict):
         )
 
 
-def get_webhook_ssm(secret_name=None):
-    stage = os.environ['DSS_DEPLOYMENT_STAGE']
-    secrets_store = os.environ['DSS_SECRETS_STORE']
-    if secret_name is None:
-        secret_name = 'monitor-webhook'
-    secret_id = f'{secrets_store}/{stage}/{secret_name}'
-    res = secretsmanager.get_secret_value(SecretId=secret_id)
-    return res['SecretString']
-
+def format_data_size(value: int):
+    base = 1000
+    bytes = float(value)
```

Comment:
`bytes` shadows a built-in: https://docs.python.org/3/library/functions.html#func-bytes .  Also `bytes` is typically an int so it should probably be renamed anyhow.

Reply:
renamed